src/audio/host-zephyr.c

```diff
@@ -34,6 +34,8 @@
 
 static const struct comp_driver comp_host;
 
+LOG_MODULE_REGISTER(host_comp, CONFIG_SOF_LOG_LEVEL);
+
```

Comment:
In retrospect, unifying with DECLARE_TR_CTX would have been nicer. Now we need the double boilerplate in each file. OTOH, that would mean more SOF specific macro magic. The explicit LOG_MODULE_REGISTER  statements are at least Zephyr-like and make the code look more standard.